.travis.yml

```diff
@@ -29,22 +29,51 @@ env:
     - SEQ_PG_PW=sequelize_test
     - SEQ_PG_HOST=127.0.0.1
     - SEQ_PG_PORT=8998
+    # oracle info
+    - NODE_ORACLEDB_HOST: localhost
+
+before_install:
+  - |
+    if [ $ORACLE_VER ]; then 
+      mkdir -p /opt/oracle && 
+      cd /opt/oracle && 
+      wget https://s3.amazonaws.com/sequelize/instantclient-basic-linux.x64-12.2.0.1.0.zip &&
```

Comment:
Instant Client 19.3 doesn't need a clickthrough from Oracle now, so the wget URL should use the Oracle official link, ie.  https://download.oracle.com/otn_software/linux/instantclient/193000/instantclient-basic-linux.x64-19.3.0.0.0dbru.zip

Also, recent node-oracledb versions don't need the sdk.

Reply:
Hi @cjbj thanks for the comment. Clearly you know much more about oracle than I do :) would you be willing to create a fork of this PR and make some updates, bringing this PR to life again?